src/repo/learnocaml_exercise.ml

```diff
@@ -21,24 +21,49 @@ type t =
     dependencies : string list;
   }
 
+
+type t =
+  | Subexercise of (id * exercise list)
```

Comment:
As a reminder: this type should be refined to include the boolean `student_hidden` and the weights (`student_weight`, `teacher_weight`)

Reply:
(as well as the boolean `check_all_against`)